aten/src/TH/generic/THTensor.cpp

```diff
@@ -527,14 +527,12 @@ int THTensor_(isSameSizeAs)(const THTensor *self, const THTensor* src)
 
 int THTensor_(isSetTo)(const THTensor *self, const THTensor* src)
 {
-  if (!THTensor_getStoragePtr(self))
```

Comment:
Won't the next `if` segfault if both storages are nullptr? 
Or are the storage supposed to never be nullptr anymore?

Reply:
They aren't supposed to be nullptr anymore (in TH).